curp/src/rpc/mod.rs

```diff
@@ -79,6 +79,17 @@ mod proto {
     }
 }
 
+impl From<Member> for PbMember {
+    fn from(member: Member) -> Self {
+        Self {
+            id: member.id(),
+            name: member.name().to_owned(),
+            addrs: member.addrs().to_vec(),
+            is_learner: member.is_learner(),
+        }
+    }
+}
```

Comment:
Move this `From` to the file that defines the `Member` struct to avoid unnecessary clone

Reply:
done